docs/sources/write/tooling-and-workflows/_index.md

```diff
@@ -143,6 +143,37 @@ Run `git fetch` before `git status` for the most accurate status.
 
 To understand the output of `git status`, refer to [Git - git-status Documentation](https://git-scm.com/docs/git-status#_output).
 
+### Checkout a PR branch from a fork
+
+> A fork is a new repository that shares code and visibility settings with the original "upstream" repository.
+
+For more information, refer to [About forks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks).
+
+To checkout a PR branch from a fork, use the `gh` tool.
```

Comment:
Should we include a link to where to obtain the gh tool?  (I've never used it before, so I don't know anything about it).

Reply:
Good point, I've added [c974289](https://github.com/grafana/writers-toolkit/pull/365/commits/c97428952d3ff8172fe991d1c3c3250b0e8ae9c6)